thread/src/main/kotlin/io/github/paulgriffith/kindling/thread/model/ThreadDump.kt

```diff
@@ -63,9 +59,19 @@ data class ThreadDump internal constructor(
             "(?<name>.*)"
             \s*CPU:\s(?<cpu>\d{1,3}\.\d{2})%
             \s*java\.lang\.Thread\.State:\s(?<state>\w+_?\w+)
-            \s*(?<stack>[\S\s]+?)[\r\n]*
+            \s*(?<stack>[\S\s]+?)[\r\n]{2,}
         """.trimIndent().toRegex(RegexOption.COMMENTS)
 
+        private val webThreadRegex = """
+        (?<isDaemon>Daemon )?Thread \[(?<name>.*)] id=(?<id>\d*), \((?<state>\w*)\)
+        ?(?<stack>\s{4}[\S\s]+?)?
+        ?(?=(?:Daemon )?Thread |")
+        """.trimIndent().toRegex()
```

Comment:
I think I prefer the actual pattern indented in one more tab stop, so it's clearly part of the same string (see line 58:63 above)